packages/api/test/server-test.js

```diff
@@ -34,10 +34,18 @@ describe('flaky express server', () => {
     const respJSON = await resp.json();
     assert.strictEqual(respJSON.message, 'goodnight moon');
   });
-  it('it returns a list of repositories, when you call GET on /repository');
+
+  it('it returns a json object with the list of repositories, when you call GET on /repos', async () => {
+    const resp = await fetch('http://localhost:3000/repos', {});
+    //TODO: get rid of this hard-coding which depends on certain collections never being edited
+    var sol = ['firstRepo', 'fourthRepo', 'secondRepo', 'thirdRepo'];
+    const respJSON = await resp.json()
+    assert.deepEqual(respJSON.repoNames, sol);
+  });
+  
   it('it returns a single repository, when you call GET on /repository/:id');
   it('it creates a repository, when you call POST on /repository');
   after(() => {
-    server.close();
+    // server.close();
```

Comment:
I think tests are never finishing because we're missing this close after all the tests.